arviz/rcparams.py

```diff
@@ -107,6 +108,29 @@ def _validate_boolean(value):
     "plot.bokeh.figure.width": (500, _validate_positive_int),
     "plot.bokeh.figure.height": (500, _validate_positive_int),
     "plot.bokeh.show": (True, _validate_boolean),
+    "plot.bokeh.marker": (
+        "Cross",
+        _make_validate_choice(
+            {
+                "Asterisk",
+                "Circle",
+                "CircleCross",
+                "CircleX",
+                "Cross",
+                "Dash",
+                "Diamond",
+                "DiamondCross",
+                "Hex",
+                "InvertedTriangle",
+                "Square",
+                "SquareCross",
+                "SquareX",
+                "Triangle",
+                "X",
+            },
+            case_sensitive=True,
+        ),
+    ),
```

Comment:
I think it will be more readable to create a `_validate_bokeh_marker = _make_validate_choice...` around line 94 and then here use this new function so everything fits in a couple of lines max.